Angular/material navbar/README.md

```diff
@@ -0,0 +1,27 @@
+# ProjectNavbar
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 12.0.1.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+
+## Code scaffolding
+
+Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+
+## Build
+
+Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
```

Comment:
Read the instructions in the project [README](https://github.com/ALPHAVIO/Mirage-UI/blob/master/README.md), you need to add a video/ gif to your component readme. Refer to this [README template](https://github.com/ALPHAVIO/Mirage-UI/blob/master/README_TEMPLATE.md) you need to follow for creating the README for your component.

Some more things required in this README other than setup instructions